CONTRIBUTING.md

```diff
@@ -34,24 +34,24 @@ The majority of new GitHub Issues will be used to add new individuals to Rest In
 
 Verify there are no duplicate Issues for this name or handle. If there are, attempt to merge their contents and apply appropriate labels.
 
-### Triaging a Pull Request
+### Triaging a Pull Request (PR)
 
-Pull requests could be anything from People, new data, code changes, web site changes, documentation or more. Applying the appropriate labels will be most important to ensuring they are handled correctly.
+Pull requests (PR) could be anything from People, new data, code changes, web site changes, documentation or more. Applying the appropriate labels will be most important to ensuring they are handled correctly.
```

Comment:
PRs could be anything from people, new data, code changes, web site changes, documentation, or more. Applying the appropriate labels will be most important to ensuring they are handled correctly.